lib/core/src/lsUtil.cpp

```diff
@@ -9,8 +9,9 @@
 #include "irods/rcGlobalExtern.h"
 #include "irods/filesystem.hpp"
 #include "irods/irods_query.hpp"
+#include "irods/escape_utilities.hpp"
 
-#include <boost/format.hpp>
+#include <fmt/format.h>
```

Comment:
From @SwooshyCueb:

> This commit might go with issue 7763

_Origin: https://github.com/irods/irods/pull/7819#discussion_r1646495022_

Reply:
Linked to issue 7763.